python/mdvtools/dbutils/routes.py

```
@@ -1,151 +1,587 @@
+                        logger.exception(f"Error extracting thumbnail for project at {project_path}: {e}")
+                        return None
+                    
+                # Create a new Project record in the database with the path
+                logger.info("Adding new project to the database")
+                new_project = ProjectService.add_new_project(path=project_path)
+
+                if new_project:
+                    # Step 5: Associate the admin user with the new project and grant all permissions
+                    if ENABLE_AUTH:
+                        current_user_id = user_data['id']
+                        user_project = UserProjectService.add_or_update_user_project(
```

Ruff (0.8.2)

Local variable `user_project` is assigned to but never used

Remove assignment to unused variable `user_project`

(F841)